engine/apps/slack/tests/test_slack_client.py

```diff
@@ -167,14 +167,33 @@ def test_slack_client_ratelimit(monkeypatch, error, make_organization_with_slack
     _, slack_team_identity = make_organization_with_slack_team_identity()
     client = SlackClient(slack_team_identity)
 
+    return_value = {"status": 429, "body": json.dumps({"ok": False, "error": error}), "headers": {"Retry-After": "1"}}
+    with patch(
+        "slack_sdk.web.base_client.BaseClient._perform_urllib_http_request_internal", return_value=return_value
+    ) as mock_request:
+        with pytest.raises(SlackAPIRatelimitError):
+            client.api_call("auth.test")
+
+    # no slack built-in retries by default
+    assert len(mock_request.mock_calls) == 1
+
+
+@pytest.mark.parametrize("error", ["ratelimited", "rate_limited", "message_limit_exceeded"])
+@pytest.mark.django_db
+def test_slack_client_ratelimit_enable_retry(monkeypatch, error, make_organization_with_slack_team_identity):
+    monkeypatch.undo()  # undo engine.conftest.mock_slack_api_call
```

Comment:
this only undo's the monkeypatch for this test, right?

Reply:
correct!